### PR TITLE
CocoaPods support

### PR DESCRIPTION
**Pull request by frootloops**
Carthage too complex for me. I need something simple like cacao.

### CODE REVIEW

File: Runes.podspec [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Pod::Spec.new do |spec|
+  spec.name = 'Runes'
+  spec.version = '1.1.1'
+  spec.summary = 'Indecipherable symbols that some people claim have actual meaning.'
```
Comment (gfontenot) [inline Runes.podspec]: This is funny in the README, but can we change this to "Functional operators for Swift"?

File: Runes.podspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  spec.homepage = 'https://github.com/thoughtbot/runes'
+  spec.license = { :type => 'MIT', :file => 'LICENSE' }
+  spec.author = {
+    'Gordon Fontenot' => 'gordon@thoughtbot.com',
+    'Adam Sharp' => 'adsharp@me.com',
```
Comment (gfontenot) [inline Runes.podspec]: Remove Adam as an author (although I super duper appreciate his contribution as well as your effort in trying to track down the author info)

**Comment by keith** [comment]
In the future it would be better if you didn't push the pods to trunk before submitting pull requests because then you don't have to deal with ownership issues.

**Comment by gfontenot** [comment]
Thanks! Couple notes, but then I'll get this into master. I've actually claimed ownership of the pod (thanks @Keithbsmiley), so you won't be able to update it there, so don't worry about that.

**Comment by frootloops** [comment]
Sorry for pod, I didn't thought about it.  It's my first steps in iOS world. I start working on my first project, found this brilliant article (http://robots.thoughtbot.com/parsing-embedded-json-and-arrays-in-swift) and I can't install Carthage with CocoaPods. Then I start making pod...

**Comment by gfontenot** [comment]
Thanks! Merged in ca822ad

**Comment by sharplet** [comment]
Hey lets remove @gfontenot as an author too and replace it with "Our Haskell Overlords"

**Comment by gfontenot** [comment]
That's applicable for most things that I do.

**Pull request closed without merging**